content/en/getting_started/integrations/aws.md

```diff
@@ -109,7 +109,7 @@ Before getting started, ensure you have the following prerequisites:
 2. Go to the [AWS integration configuration page][8] in Datadog and click **Add AWS Account**.
 
 3. Configure the integration's settings under the **Automatically using CloudFormation** option.  
-    a. Select the AWS regions to integrate with.  
+    a. Select the AWS region where the CloudFormation stack is created.  
```

Comment:
```suggestion
    a. Select the AWS region where the CloudFormation stack is located.  
```

Is this still correct? If so I think it is a little clearer.

Reply:
Yes, that is good.

Reply:
If you commit the suggestion, I can approve. Thanks!

Reply:
Reminder to commit this :)